lib/commands/draft.js

```diff
@@ -52,30 +53,32 @@ module.exports = async function (args) {
     toCommit = await git.getLastTagCommit(tagPattern)
     if (!toCommit) { toCommit = 'HEAD' }
   }
-  const commits = await git.getLogs(args.fromCommit, toCommit)
-  logger.debug('Found %d commits from %s to %s', commits.length, args.fromCommit, toCommit)
+  if (!args.ghReleaseBody) {
+    const commits = await git.getLogs(args.fromCommit, toCommit)
+    logger.debug('Found %d commits from %s to %s', commits.length, args.fromCommit, toCommit)
 
-  const commitsPR = commits.map(addPullRequestId)
-  const commitsWithPR = commitsPR.filter(c => c.pullRequest.id != null)
-  const commitsWithoutPR = commitsPR.filter(c => c.pullRequest.id == null)
-  logger.debug('There are %d commits message that contain a PR id and %d don\'t', commitsWithPR.length, commitsWithoutPR.length)
+    const commitsPR = commits.map(addPullRequestId)
```

Comment:
I would move this outside of the `if`.

The `throw new Error('There are ZERO commit to release!')` error is useful

Reply:
I ended up removing the `if` since the populate `addLines` method was in if a little more downstream. Now err should be able to be thrown